pallets/foreign-investments/src/lib.rs

```diff
@@ -105,7 +105,11 @@ pub mod pallet {
 
 	use super::*;
 
+	/// The current storage version.
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
+
 	#[pallet::pallet]
+	#[pallet::storage_version(STORAGE_VERSION)]
```

Comment:
This slipped under our radar. Note to ourselves for not forgetting storage versions of pallets.

Reply:
Good eye!